source/speech/speech.py

```diff
@@ -1514,7 +1541,7 @@ def getTextInfoSpeech(  # noqa: C901
 	reportIndentation = (
 		unit == textInfos.UNIT_LINE and formatConfig["reportLineIndentation"] != ReportLineIndentation.OFF
 	)
-	# For performance reasons, when navigating by paragraph or table cell, spelling errors will not be announced.
+	# For performance reasons, when navigating by paragraph or table cell, spelling errors and language description will not be announced.
```

Comment:
Have you personally experimented these performance issues during your tests? If not, why this choice?

Reply:
IMO these are two different tasks.
There is no point ignoring language reporting if this does not cause any perf issue.